index.html

```diff
@@ -1020,6 +1165,70 @@ <h2>
         </dd>
       </dl>
     </section>
+    <section data-dfn-for="GamepadTouch">
+      <h2>
+        <dfn>GamepadTouch</dfn> Interface
+      </h2>
+      <p>
+        This interface defines a single touch event on a gamepad device that
+        supports input. The event consists of a touch id that uniquely
+        identifies the touch point from the time the input medium (e.g. finger,
+        stylus, etc) makes contact with the touch device, up to the time the
+        input medium is no longer making contact with the touch device.
+      </p>
+      <pre class="idl" data-cite="geometry-1">
+        [Exposed=Window, SecureContext]
+        interface GamepadTouch {
+          readonly attribute unsigned long touchId;
+          readonly attribute octet surfaceId;
+          readonly attribute DOMPoint position;
+          readonly attribute DOMRect? surfaceDimensions;
+        };
+      </pre>
+      <dl>
+        <dt>
+          <dfn>touchId</dfn> attribute
+        </dt>
+        <dd>
+          Unique id of the touch event. Range is [0, 4294967295].
+          <p>
+            The user agent is responsible for:
+          </p>
+          <ul>
+            <li>Incrementing the id for each subsequent touch event based on
+            information provided by the device API.
+            </li>
+            <li>Maintaining the uniqueness of the id per origin to prevent
+            fingerprinting.
+            </li>
+          </ul>{{GamepadTouch/touchId}} SHOULD be set to a default value of 0
```

Comment:
Instead of using `0` here, should we store a `touchId` counter in an internal slot somewhere? Maybe in the target `Gamepad` object?